dashboards-observability/public/components/visualizations/charts/helpers/viz_types.ts

```diff
@@ -43,10 +57,18 @@ export const getVizContainerProps = ({
   indexFields = {},
   userConfigs = {},
   appData = {},
+  explorer = { explorerData: { jsonData: [], jsonDataAll: [] } },
 }: IVizContainerProps): IVisualizationContainerProps => {
-  const visType = {
-    ...getVisType(vizId),
-  }
+  const getVisTypeData = () => {
+    if (vizId === visChartTypes.Line || vizId === visChartTypes.Scatter) {
+      return vizId === visChartTypes.Line
+        ? { ...getVisType(vizId, { type: visChartTypes.Line }) }
+        : { ...getVisType(vizId, { type: visChartTypes.Scatter }) };
```

Comment:
Similar to one of the comments given in another file. Please check how the code can be reduced here and with better readability.